src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -754,6 +754,16 @@ internal static partial class Library
                     returnBehavior: ReturnBehavior.ReturnFalseIfAnyArgIsBlank,
                     targetFunction: StartsWith)
             },
+            {
+                BuiltinFunctionsCore.StringInterpolation,
+                StandardErrorHandling<StringValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: ReplaceBlankWithEmptyString,
+                    checkRuntimeTypes: ExactValueType<StringValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
+                    targetFunction: Concatenate)
```

Comment:
>Concatenate [](http://example.com/codeflow?start=36&length=11)

This is subtle: we introduce a new function for type-checking purposes, but it executes the same way as concatenate.